src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs

```diff
@@ -166,6 +166,10 @@ protected virtual unsafe IntPtr AppWndProc(IntPtr hWnd, uint msg, IntPtr wParam,
                 case WindowsMessage.WM_MBUTTONDOWN:
                 case WindowsMessage.WM_XBUTTONDOWN:
                     {
+                        if (Win8Plus)
```

Comment:
This should be a feature flag from Win32PlatformOptions that also checks for the Windows version on startup